src/components/screens/EventRecord.js

```diff
@@ -154,7 +156,7 @@ export const EventRecord = ({ data, navigation }) => {
         {!!priceInformations && !!priceInformations.length && !!priceInformations[0].description && (
           <View>
             <TitleContainer>
-              <Title accessibilityLabel={`${texts.eventRecord.prices} (Überschrift)`}>
+              <Title accessibilityLabel={`${texts.eventRecord.prices} ${a11yText.heading}`}>
```

Comment:
please double check spoken breaks everywhere with `(...)` for strings with "Taste" or "Überschrift"

Reply:
done in 71c498a949d6f531126e672bc262773d65a2cbf2

Reply:
how did you check it? did you need to do changes?